pubspec.yaml

```diff
@@ -44,9 +44,8 @@ flutter:
   uses-material-design: true
 
   # To add assets to your application, add an assets section, like this:
-  # assets:
-  #   - images/a_dot_burr.jpeg
-  #   - images/a_dot_ham.jpeg
+  assets:
+    - assets/
```

Comment:
we don't have fonts?

Reply:
ah I wasn't instructed about the fonts so wala  pa. Should I add it to my next push?